modules/nw-ingress-setting-thread-count.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-setting-thread-count_{context}"]
+= Setting Ingress Controller thread count
+A cluster administrator can set the thread count to increase the amount of incoming connections a cluster can handle. You can patch an existing Ingress Controller to increase the amount of threads.
+
+. Procedure
+The following assumes that you already created an Ingress Controller.
+
+.. Update the Ingress Controller to increase the number of threads
```

Comment:
`..` will give you a second-level ordered list item, but this is a first-level step. Normally, you would need change it to `.`. However, in this case, this is a single-step procedure, so it should be an unordered list item (`*`) instead.

Reply:
This is def a copy/paste error on my part. I copied the structure of this procedure from a 2-step. My b...need to be more careful.